### ISSUE TITLE
webrtc推流不能被控制台检测到,不通过回调接口,视频不能保存

### ISSUE THREAD

**Issue by ziyue7575** [ossrs/srs#2073]
**描述(Description)**
webrtc推流不能被控制台检测到,不通过回调接口,视频不能保存(webrtc可以拉流)

1. SRS版本(Version): `4.0.56`
1. SRS的配置如下(Config):
```
listen              1935;
max_connections     1000;
srs_log_file        /home/turnuser/srs/trunk/objs/srs.log;
#srs_log_tank        console;
#daemon              off;

http_server {
    enabled         on;
    listen          8080;
    dir             ./objs/nginx/html;
    https {
        enabled on;
        listen 8088;
        key ./conf/top_key.key;
        cert ./conf/top_chain.crt;
    }
}
http_api {
    enabled         on;
    listen          1985;
    https {
        enabled on;
        listen 1990;
        key ./conf/top_key.key;
        cert ./conf/top_chain.crt;
    }
    raw_api {
        enabled             on;
        allow_reload        on;
        allow_query         on;
        allow_update        on;
    }
}
stats {
    network         0;
}
rtc_server {
    enabled         on;
    # Listen at udp://8000
    listen          8000;
    #
    candidate       服务器ip;
}
vhost __defaultVhost__ {
    # 0延迟
     min_latency     on;
     play {
         mw_latency      0;
         mw_msgs         0;
     }
    http_remux {
        enabled     on;
        mount       [vhost]/[app]/[stream].flv;
    }
     hls {
         enabled         on;
         hls_path        /home/turnuser/srs/trunk/objs/nginx/html;
         hls_fragment    10;
         hls_window      60;
     }
     # 保存视频
     dvr {
        enabled      on;
        # 这个里的都保存
        dvr_apply       all;
        dvr_path        /opt/livestream/temp/[app]/[stream]/[timestamp].flv;
        dvr_plan        session;
        time_jitter     full;
    }
    rtc {
        enabled     on;
        bframe      discard;
    }

}
```

**Comment by winlinvip** [comment]
Known issue, WebRTC is still under development and has not been released yet. Welcome to contribute patches.

`TRANS_BY_GPT3`

**Comment by winlinvip** [comment]
Fixed by #2509
